vendor-extra/LiberoPatternsBundle/src/Resources/translations/messages+intl-icu.en.xlf

```diff
@@ -24,6 +24,30 @@
                 <source>Publication date</source>
             </trans-unit>
 
+            <trans-unit id="5" resname="libero.patterns.teaser.categories.label">
+                <source>Categories that include this item</source>
+            </trans-unit>
+
+            <trans-unit id="6" resname="libero.patterns.teaser.meta.label">
+                <source>Publication details</source>
+            </trans-unit>
+
+            <trans-unit id="7" resname="libero.patterns.teaser.meta.type.label">
+                <source>Type of article</source>
+            </trans-unit>
+
+            <trans-unit id="8" resname="libero.patterns.teaser.meta.date.label">
+                <source>Publication date</source>
+            </trans-unit>
+
+            <trans-unit id="9" resname="libero.patterns.teaser.authors">
+                <source>{ count, plural,
+                    =1 {{author1}}
+                    =2 {{author1}, {author2}}
+                    other {{author1} et al.}
+                }</source>
```

Comment:
Possibly unusual, but flexible, format. Made more complicated as the replacements (eg `{author1}`) aren't (necessarily) strings.